.github/ISSUE_TEMPLATE/enhancement.yml

```
@@ -0,0 +1,51 @@
+# Copyright 2022 The Engula Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Enhancement
+description: Enhance code, document, and more
+labels: [ "enhancement" ]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you very much for your good enhancement for Engula!
```

```suggestion
        Thank you very much for your enhancement for Engula!
```

"Good" and "enhancement" seem redundant.